project_template/package.json

```diff
@@ -12,11 +12,22 @@
     "prettier": "prettier '{,!(node_modules|cordova)/**/}*.js'",
     "lint": "yarn run eslint && yarn run stylelint",
     "eslint": "eslint src/ test/",
-    "stylelint": "stylelint 'src/**/*.scss'"
+    "stylelint": "stylelint 'src/**/*.scss'",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "prettier --write",
```

Comment:
we should probably lint with `eslint --fix`, which will run prettier behind the scenes anyway (see `eslint-plugin-prettier`)

Reply:
Yep, but also here let's do it on a separate PR this just maintains the old behavior.